community/foundation-delegation-program.md

```diff
@@ -46,7 +46,7 @@ During this period, so long as the validator maintains high uptime and does not
 The minimum requirements for participation in the program are as follows:
 
 * Run an active mainnet validator **or** an active Mocha testnet validator for at least 1 month before application deadline
-* Run a bridge node (on mainnet if you are already an active mainnet validator or on Mocha testnet if not) that is connected and reporting to the Celestia Labs [OTEL collector](https://docs.celestia.org/nodes/celestia-node-metrics) (for new applicants - on testnet, so that we can evaluate performance)
+* Run a bridge node (on mainnet if you are already an active mainnet validator or on Mocha testnet if not) that is connected and reporting to the Celestia Labs [OTEL collector](../nodes/celestia-node-metrics) (for new applicants - on testnet, so that we can evaluate performance)
```

Comment:
```suggestion
* Run a bridge node (on mainnet if you are already an active mainnet validator or on Mocha testnet if not) that is connected and reporting to the Celestia Labs [OTEL collector](../nodes/celestia-node-metrics.md) (for new applicants - on testnet, so that we can evaluate performance)
```